presto-spi/src/main/java/com/facebook/presto/spi/block/LazyBlockEncoding.java

```diff
@@ -21,7 +21,7 @@
 public class LazyBlockEncoding
         implements BlockEncoding
 {
-    private static final String NAME = "LAZY";
+    public static final String NAME = "LAZY";
```

Comment:
There is already an encoding above with this name

Reply:
`GroupByIdBlockEncoding` is not used. Deleting it.